tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -420,22 +432,29 @@ function plot_variational_marginals(z, sym2range)
         if sum(length.(indices)) > 1
             offset = 1
             for r in indices
-                for j in r
-                    p = density(
-                        z[j, :]; title="$(sym)[$offset]", titlefontsize=10, label=""
-                    )
-                    push!(ps, p)
-
-                    offset += 1
-                end
+                p = density(
+                    z[r, :];
+                    title="$(sym)[$offset]",
+                    titlefontsize=10,
+                    label="",
+                    margin=2.0mm,
```

Comment:
I'm a bit worried if the `margin` settings will be correct for all output formats (in HTML, markdown etc.).

Reply:
@devmotion Yes it will have to be tested. Locally `margin=2.00mm` looks good, but once rendered as HTML it normally requires `margin=6.00mm`, as found in the pPCA tutorial.
I just changed it to be `margin=6.00mm` to see how it works - maybe revert it later.